### DISCUSSION TITLE
Some doubts about Links vs. Tags and how to use them

### DISCUSSION THREAD

**Discussion by rodrigo-rufino** [BryanJenksCommunity/FAQ#97] [Q&A] [answered]
I've been watching your comprehensive guide and you said you only links as tags. But I'm not able to translate some of my tags into links. Gor example:
- #gardening: All the things related to gardening.
- #gardening/species: All the species I have catalogued.
- #gardening/species/<vegetables | ornamental | aromatic>: Some species categories.

How would you translate these tags and subtags into links? And if I would colour each one of them with a different colour in my graph, how would I do it?

**Comment by tallguyjenks** [comment, marked as answer]
the difference is that what you listed as tags is the actual items not a status.

i use nested tags like `#gardening/species/vegetables` like `#input/book/processing` but I use emoji so its `#📥️/📚️/🟧️` 

you're using tags as tags in their intended form. i use tags to denote status and treat [[links]] as both [[links]] and [[tags]]. 

because your nested tags are not statuses, rather they are actually related items if it were my notes I would make mocs for each level.

[[Gardening]] contains links to [[species]]
[[species]] contains links to [[vegetables]], [[ornamental]], and [[aromatic]] and each of those links to a note for each plant etc.

Tags as status, links as tags, and where there was nested tags, replace with MOC (map of content) structure 

at least that's my approach

**Reply by rodrigo-rufino** [reply]
Nice, it makes sense. But if you would give different colours to each type of plant, how would you do using only links? With tags, it can be easily done, but I'm not able to find a way to do the same with graphs.

**Reply by tallguyjenks** [reply]
could do that with the graph groupings i guess because you could use a regex search for each type and the link to that type in the doc using regex and then with a graph group query then assign a color to the result set
